modules/ephemeral-storage-csi-inline-overview-admin-plugin.adoc

```diff
@@ -0,0 +1,106 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/ephemeral-storage-csi-inline.adoc
+
+:_content-type: CONCEPT
+[id="ephemeral-storage-csi-overview-admin-plugin_{context}"]
+= CSI Volume Admission plug-in
+
+The CSI Volume Admission plug-in provides a mechanism where the use of an individual CSI driver capable of provisioning CSI ephemeral volumes can be restricted on pod admission. Administrators or distributions can optionally add a `csi-ephemeral-volume-profile` label to a `CSIDriver` object, and the label is then inspected by the Admission plug-in and used in enforcement, warning, and audit decisions.
+
+== Overview
+
+To make use of the CSI Volume Admission plug-in, administrators or distributions can add the `security.openshift.io/csi-ephemeral-volume-profile` label, which declares the CSI driver’s effective pod security profile when it is used to provide CSI ephemeral volumes. This “effective profile” communicates that a pod can use the CSI driver to mount CSI ephemeral volumes when the pod’s namespace is governed by a pod security standard:
+
+[source, yaml]
+----
+kind: CSIDriver
+metadata:
+  name: csi.mydriver.company.org
+  labels:
+    security.openshift.io/csi-ephemeral-volume-profile: restricted
+----
+
+The CSI Volume Admission plug-in inspects pod volumes when pods are created (existing pods that use CSI volumes are not affected). If a pod uses a container storage interface (CSI) volume, the plug-in looks up the `CSIDriver` object and inspects the `csi-ephemeral-volume-profile` label, and then use the label’s value in its xref:../../storage/container_storage_interface/ephemeral-storage-csi-inline.adoc#security-profile-enforcement[enforcement], xref:../../storage/container_storage_interface/ephemeral-storage-csi-inline.adoc#security-profile-warning[warning], and xref:../../storage/container_storage_interface/ephemeral-storage-csi-inline.adoc#security-profile-audit[audit] decisions. 
```

Comment:
```suggestion
The CSI Volume Admission plug-in inspects pod volumes when pods are created (existing pods that use CSI volumes are not affected). If a pod uses a container storage interface (CSI) volume, the plug-in looks up the `CSIDriver` object and inspects the `csi-ephemeral-volume-profile` label, and then uses the label’s value in its xref:../../storage/container_storage_interface/ephemeral-storage-csi-inline.adoc#security-profile-enforcement[enforcement], xref:../../storage/container_storage_interface/ephemeral-storage-csi-inline.adoc#security-profile-warning[warning], and xref:../../storage/container_storage_interface/ephemeral-storage-csi-inline.adoc#security-profile-audit[audit] decisions. 
```

Reply:
[xrefs are not allowed in modules](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-hyperlinks-and-cross-references) - I think you need to refer readers to the "Additional resources" section here for more info on these three things, and then list those links in the assembly "Additional resources" section.